scripts/loadable_app/hello.c

```diff
@@ -0,0 +1,12 @@
+int a = 9;
+int b = 9;
+
+int x()
+{
+return 7;
+}
+int main(int argc, char **argv)
+{
+	int p = x();
+	return a + p + x();
```

Comment:
I've tried the new suggested command-line, works!

I think it's OK to have `-nostdlib`. This should to pervent taking C standard library from host compiler (and for host target). That lib is unlikely to be compatible with Embox

However, new command line is not in `loadable_app.sh`. Could you change it to new one? 
This is a first issue preventing merging